api/src/opentrons/protocol_engine/commands/set_rail_lights.py

```diff
@@ -0,0 +1,60 @@
+"""setRailLights command request, result, and implementation models."""
+from __future__ import annotations
+from pydantic import BaseModel, Field
+from typing import TYPE_CHECKING, Optional, Type
+from typing_extensions import Literal
+
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+
+if TYPE_CHECKING:
+    from ..execution import RailLightsHandler
+
+SetRailLightsCommandType = Literal["setRailLights"]
+
+
+class SetRailLightsParams(BaseModel):
+    """Payload required to set the rail lights on or off."""
+
+    on: bool = Field(
+        True,
+        description="The field that determines if the light is turned off or on.",
+    )
```

Comment:
```suggestion
    on: bool = Field(
        ...,
        description="The field that determines if the light is turned off or on.",
    )
```

In my (soft) opinion, we should remove the default here.